src/azure-cli/azure/cli/command_modules/acr/commands.py

```diff
@@ -420,7 +420,7 @@ def load_command_table(self, _):
         g.show_command('show', 'show_encryption')
         g.command('rotate-key', "rotate_key")
 
-    with self.command_group('acr connected-registry', acr_connected_registry_util, is_preview=True) as g:
+    with self.command_group('acr connected-registry', acr_connected_registry_util) as g:
```

Comment:
Is it change expected?

Reply:
Good catch, please do not add any unrelated logic to this PR, thanks. @msyyc

Reply:
If you think it is necessary to remove the `is_preview` flag, could you submit a separate PR to remove the `is_preview=True` flag ? @huanwu @trisavo-msft

Reply:
This change should be merged by this PR https://github.com/Azure/azure-cli/pull/31475/files